django_x509/tests/test_ca.py

```diff
@@ -467,8 +467,12 @@ def test_import_ca_cert_validation_error(self):
             ca.private_key = private_key
             ca.full_clean()
         except ValidationError as e:
-            self.assertIn("('PEM routines', 'PEM_read_bio', 'no start line')",
-                          str(e.message_dict['certificate'][0]))
+            error_msg = str(e.message_dict['certificate'][0])
+            self.assertTrue(
+                "('PEM routines', 'PEM_read_bio', 'no start line')" in error_msg
```

Comment:
Maybe this can be avoided if we're only interested in `cryptography>=2.6.0`. I left it as you might still depend on 2.4 elsewhere. Let me know if you want to support 2.4 and 2.6.

Reply:
is good to keep backward compatibility for a while, but I would add a comment explaining why this is being done, so we won't be puzzled when looking at this in the future, can you do that please?